assets/css/admin-post-setting-style.css

```diff
@@ -0,0 +1,329 @@
+#post-body-content #titlediv #title-prompt-text {
```

Comment:
May I recommend moving this file to scss and wrapping the a class around the whole thing.

Reply:
I have pushed code to convert this file to SCSS to help do this

Reply:
Done